### ISSUE TITLE
<head> elements duplicated with nix-built exe (and not with `ob run`)

### ISSUE THREAD

**Issue by charlieshanley** [obsidiansystems/obelisk#428]
Hello,

When linking to bootstrap css and also using my own css file in "static/", several elements in \<head\> end up duplicated, and the link to my css file is removed. This issue presents when running the nix-built executable, and does not present with `ob run`.

A minimal example is [here](https://github.com/charlieshanley/reprex), with some crucial bits reproduced below.

Here's Frontend.hs:
```hs
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE OverloadedStrings #-}
{-# LANGUAGE TypeApplications #-}
module Frontend where

import qualified Data.Text as T
import Obelisk.Frontend
import Obelisk.Route
import Reflex.Dom.Core

import Common.Api
import Common.Route
import Obelisk.Generated.Static

frontend :: Frontend (R FrontendRoute)
frontend = Frontend
  { _frontend_head = do
      el "title" $ text "Obelisk Minimal Example"
      let bootstrap = "rel" =: "stylesheet"
                   <> "href" =: "https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
                   <> "integrity" =: "sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
                   <> "crossorigin" =: "anonymous"
          myCss = "rel" =: "stylesheet"
               <> "href" =: (static @"styles.css")
      elAttr "link" bootstrap blank
      elAttr "link" myCss blank
  , _frontend_body = do
      text "Welcome to Obelisk!"
      el "p" $ text $ T.pack commonStuff
      elAttr "img" ("src" =: static @"obelisk.jpg") blank
  }
```
Here's the resulting dom with `ob run`:
![image](https://user-images.githubusercontent.com/8228888/57463821-8ae5b600-7249-11e9-8ee1-82025d38e2ae.png)

And here's the resulting dom with `nix-build -A exe -o result-exe`, `cd result-exe`, `./backend`:
![image](https://user-images.githubusercontent.com/8228888/57463439-cf248680-7248-11e9-96cf-4499be5d74df.png)
The title and bootstrap link occur twice, and my css link doesn't occur.

I think this is related to #356, but am not sure how.

**Comment by tomsmalley** [comment]
Hmm the duplication smells like a bug in hydration, rather annoying that it's GHCJS only - test suite won't have caught it. You are right, it is related to that issue, or more specifically https://github.com/obsidiansystems/obelisk/issues/375 which should be fixed by https://github.com/obsidiansystems/obelisk/pull/402. Regardless, even if hydration fails due to that bug, the elements should not be duplicated.

**Comment by ali-abrar** [comment]
Should be fixed now that #402 is merged. Please let us know.

**Comment by charlieshanley** [comment]
Yes, I believe it's fixed. Thanks!
